doc/whats_new/upcoming_changes/sklearn.datasets/30196.enhancement.rst

```diff
@@ -0,0 +1,6 @@
+New parameter return_X_y added to
+:func:`datasets.make_classification`. The
+default value of the parameter does not
+change how the function behaves.
+By :user:`Success Moses <SuccessMoses>` and
+:user:`Adam Cooper <arc12>`
```

Comment:
```suggestion
- New parameter ``return_X_y`` added to :func:`datasets.make_classification`. The
  default value of the parameter does not change how the function behaves.
  By :user:`Success Moses <SuccessMoses>` and :user:`Adam Cooper <arc12>`
```

Is Adam helping you in these PRs? I don't see a record of the user in the PR.

Reply:
> Is Adam helping you in these PRs? I don't see a record of the user in the PR

Adam raised the issue #16532

Reply:
I see. We can keep it, but in almost all changelog entries, only the author(s) of the PR are included. I don't mind including Adam here though.